packages/core/src/cli/eslint.ts

```diff
@@ -0,0 +1,19 @@
+import { Abort } from './error'
+import { exec } from './system'
+import { findUp, dirname } from './path'
+import { fileURLToPath } from 'url'
+
+export const ESLintNotFoundError = new Abort('Could not locate ESLint', {
```

Comment:
Similar comment here regarding the error, instead of initializing it and keeping it in memory in case we need it, I'd turn it into a function.